hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -3153,5 +3199,42 @@
       stage: V2
       tests: ["chip_sw_pwrmgr_deep_sleep_all_wake_ups"]
     }
+
```

Comment:
@moidx 
@timothytrippel 
@alphan 
@cindychip 
@weicaiyang 

let me know if this makes sense to you guys. The last one I tried to structure it in a way that I think fits into our flow today, but I could be way off.  Let me know what you think.

Reply:
It does but this test assumes that `ROM_EXEC_EN` is already enabled. (1) We should probably mention it in the testpoint (2) do we have a testpoint for that?

Reply:
ah i didn't have the `ROM_EXEC_EN` in directly because that bit didn't have a direct hardware impact. 
But as we discussed in the doc..maybe it makes sense to add.

Reply:
updated. I now changed the test case to mention the `ROM_EXEC_EN` thing.